Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -633,8 +624,7 @@ static ErrorOr<void> read_start_of_scan(AK::SeekableStream& stream, JPEGLoadingC
         return Error::from_string_literal("SOS found before reading a SOF");
     }
 
-    u16 bytes_to_read = TRY(stream.read_value<BigEndian<u16>>()) - 2;
-    TRY(ensure_bounds_okay(TRY(stream.tell()), bytes_to_read, TRY(stream.size())));
+    [[maybe_unused]] u16 const bytes_to_read = TRY(stream.read_value<BigEndian<u16>>()) - 2;
```

Comment:
why assign this if it's unused?

Reply:
Having `TRY(stream.read_value<BigEndian<u16>>())` on its own doesn't give the explanation on why we are doing it. And I would rather keep the name of the variable than adding a comment.